worker.py

```diff
@@ -742,6 +744,21 @@ def accurate_speed(self, point):
         self.update_accounts_dict(auth=False)
         return pokemon_seen + forts_seen + points_seen
 
+    def smart_throttle(self, requests=1):
+        if not config.SMART_THROTTLE:
+            return True
+
+        try:
+            # https://en.wikipedia.org/wiki/Linear_equation#Two_variables
+            # e.g. left_hashes > 2.25*left_seconds+7.5, spare = 0.05, max = 150
+            spare = config.SMART_THROTTLE * HashServer.status['maximum']
+            left_hashes = HashServer.status['remaining'] - requests
+            usable_per_second = (HashServer.status['maximum'] - spare) / 60
+            left_seconds = HashServer.status['period'] - time()
+            return left_hashes > usable_per_second * left_seconds + spare
```

Comment:
Could you move the word left in these to the end (`seconds_left`, `hashes_left`). That's generally the order they'd be spoken in (unless talking about left and right directions).